companion/lib/Controls/Fragments/FragmentActions.js

```diff
@@ -42,6 +42,12 @@ export default class FragmentActions extends CoreBase {
 	 */
 	options
 
+	/**
+	 * @type {string | undefined}
+	 * @access public
+	 */
+	name
```

Comment:
I wonder if this would make more sense as a property inside of the `options` object above?
I think that would solve the issue of it being forgotten. The real issue with that is that the loading flow for properties on this class is non-obvious, and it happens https://github.com/bitfocus/companion/blob/da425be3a3ba07c0d543220149edde7d2d58c0ab/companion/lib/Controls/ControlTypes/Button/Normal.js#L602. The other solution to it not being loaded at startup would be to add a similar loading line there